app/src/renderer/components/staking/LiDelegate.vue

```diff
@@ -116,23 +137,68 @@ export default {
 
 .li-delegate__values
   display flex
-  height 3rem
+  height 5rem
+  padding 12px 1em
+  background-color var(--app-nav)
+
+  & > .li-delegate__value:not(:first-of-type) span
```

Comment:
highly impressive css 👏 🥇

Reply:
be wary of `:first-of-type` it only refers to native elements, not classes!!!